osu.Game/Graphics/OsuFont.cs

```diff
@@ -65,16 +70,7 @@ public static string GetWeightString(Typeface typeface, FontWeight weight)
         /// <param name="family">The family string.</param>
         /// <param name="weight">The <see cref="FontWeight"/>.</param>
         /// <returns>The string representation of <paramref name="weight"/> in the specified <paramref name="family"/>.</returns>
-        public static string GetWeightString(string family, FontWeight weight)
-        {
-            string weightString = weight.ToString();
-
-            // Only exo has an explicit "regular" weight, other fonts do not
-            if (family != GetFamilyString(Typeface.Exo) && weight == FontWeight.Regular)
```

Comment:
Have you checked that this isn’t required still? I’m fairly certain this breaks Venera.

Reply:
aha, didn't see that case, looks bad. we probably want to rename this and remove "medium" altogether (or standardise to regular).

Reply:
Turns out all the naming for venera was wrong (and one instance existed without a weight suffix). Should be better aligned now.